### PR TITLE
Updated the logo to the latest, including The GO.

### PR DESCRIPTION
**Pull request by tarkatronic**
Upon request from @phillipescobar I've updated the logo in the navigation. I had to do a bit of tweaking to make it look "right". This seemed to look the best, to my non-designer eyes. Additional tweaks are welcome!

## Before

Screen Shot 2020-01-29 at 1 44 00 PM

## After

Screen Shot 2020-01-29 at 1 43 20 PM

Note that the top black bar is ever so slightly taller now.

Coincidentally, this brings it closer to the height of the top bar on the main website. That is 36px high; this is now 34px high.

### CODE REVIEW

**Comment by tarkatronic** [comment]
@chrisvogt  Any change in indentation is just a part of the logo :-/ I didn't add any padding or anything. Just copy-pasta from the svg file straight out of our asset manager. Happy to try to tweak it to the left though!

**Comment by chrisvogt** [comment]
Gotcha. Let me know if it gives you trouble and I'm happy to help out later today. I have Illustrator, which helps when editing SVG files visually. I _think_ just updating the viewbox value could cut the whitespace out.

**Comment by tarkatronic** [comment]
I honestly don't know the first thing about that, and just used the viewbox already in the brand SVG. It sounds like you have more expertise here, so I would LOVE some help. 😄

File: _includes/component/navigation.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
....85,0,0,1-3.92-3.92V192.39c-9.09,11.48-24.9,18.71-42.78,18.71-33.58,0-59.82-25.26-59.82-61.22,0-37.37,28.33-63.13,64.12-63.13,26.42,0,47.54,11.36,56.92,34.21a5.21,5.21,0,0,1,.43,1.91c0,1.08-.71,1.9-3,2.69L472.82,132a4.29,4.29,0,0,1-3.16.05,4.84,4.84,0,0,1-2-2.29c-6-11.41-16.68-19.16-33.12-19.16-21.32,0-36.59,16.65-36.59,38.24,0,20.93,13,38.14,37.2,38.14,12.73,0,22.88-6,27.61-12.59h-15.1a3.86,3.86,0,0,1-3.92-3.92V157.58a3.86,3.86,0,0,1,3.92-3.92h44.25A3.73,3.73,0,0,1,495.83,157.41Zm598.64-49.82v-1.15a.33.33,0,0,1,.08-.22.32.32,0,0,1,.21-.08h7.31a.29.29,0,0,1,.21.08.33.33,0,0,1,.08.22v1.15a.32.32,0,0,1-.08.21.29.29,0,0,1-.21.08h-2.67v7.21a.31.31,0,0,1-.3.3h-1.38a.27.27,0,0,1-.2-.09.29.29,0,0,1-.09-.21v-7.21h-2.67a.27.27,0,0,1-.29-.29Zm12.78-1.19,2,4.63,2-4.63a.41.41,0,0,1,.16-.2.54.54,0,0,1,.27-.06H1114a.27.27,0,0,1,.29.28v8.68a.27.27,0,0,1-.28.29h-1.3a.29.29,0,0,1-.21-.08.26.26,0,0,1-.08-.21v-6.55l-2.15,4.76a.52.52,0,0,1-.16.23.49.49,0,0,1-.27.06h-1.06a.48.48,0,0,1-.26-.06.45.45,0,0,1-.16-.23l-2.14-4.76v6.55a.32.32,0,0,1-.08.21.32.32,0,0,1-.2.08h-1.33a.26.26,0,0,1-.19-.08.26.26,0,0,1-.08-.21v-8.68a.27.27,0,0,1,.29-.28h2.25a.53.53,0,0,1,.26.06A.41.41,0,0,1,1107.25,106.4Z"/></g></svg>
```
Comment (chrisvogt) [inline _includes/component/navigation.html]: I don't have access to push to your PR. Here you are...

### Screenshot

Screen Shot 2020-02-03 at 9 42 53 AM

### Change

```suggestion
<svg class="logo-mark logo-mark-white py-1" xmlns="http://www.w3.org/2000/svg" height="30" viewBox="0 0 103 21.13"><path d="M20.47,1C18-.57,14.75-.21,11.86,1.61,9-.21,5.72-.57,3.26,1c-3.89,2.43-4.37,8.7-1,14,2.44,3.91,6.26,6.2,9.65,6.16s7.22-2.25,9.66-6.16C24.83,9.67,24.36,3.4,20.47,1ZM4,13.85a12.42,12.42,0,0,1-1.53-3.5,9.67,9.67,0,0,1-.33-3.3A5.27,5.27,0,0,1,4.38,2.76a5.34,5.34,0,0,1,4.85-.14c.27.11.54.24.81.38A15,15,0,0,0,7.42,6.16a12.87,12.87,0,0,0-1.93,9.65A12.35,12.35,0,0,1,4,13.85Zm17.26-3.5a12.42,12.42,0,0,1-1.53,3.5,12.35,12.35,0,0,1-1.49,2A12.59,12.59,0,0,0,16.9,7.18.39.39,0,0,0,16.38,7l0,0-4.79,3a.39.39,0,0,0-.13.54h0l.71,1.13a.39.39,0,0,0,.53.13h0l3.11-1.94c.1.3.2.61.27.91a9.61,9.61,0,0,1,.33,3.29,5.29,5.29,0,0,1-2.24,4.3,4.37,4.37,0,0,1-2.22.64h-.1a4.37,4.37,0,0,1-2.23-.64,5.32,5.32,0,0,1-2.24-4.3,10,10,0,0,1,.33-3.3,13.12,13.12,0,0,1,4-6.41A10.13,10.13,0,0,1,14.5,2.62a5.32,5.32,0,0,1,4.85.13,5.33,5.33,0,0,1,2.24,4.3A10.19,10.19,0,0,1,21.26,10.35ZM46,8a4.45,4.45,0,1,0,4.58,4.48A4.5,4.5,0,0,0,46,8Zm0,6.56a2.11,2.11,0,0,1,0-4.22,2.12,2.12,0,0,1,0,4.23Zm10.27-10H51.65a.36.36,0,0,0-.38.36V16.27a.37.37,0,0,0,.35.4h4.67c3.71,0,6.29-2.47,6.29-6.06S60,4.58,56.29,4.58Zm.11,9.7H53.86V7H56.4a3.42,3.42,0,0,1,3.47,3.64A3.47,3.47,0,0,1,56.4,14.28Zm15.42-6H70.08a.41.41,0,0,0-.37.38v.52A2.78,2.78,0,0,0,67.3,8a4.25,4.25,0,0,0-4.22,4.44,4.27,4.27,0,0,0,4.2,4.46,2.82,2.82,0,0,0,2.44-1.14v.53a.37.37,0,0,0,.36.37h1.74a.37.37,0,0,0,.37-.35V8.63a.38.38,0,0,0-.36-.38Zm-4.06,6.34a2,2,0,0,1-2-2.12,2,2,0,1,1,4,0,2,2,0,0,1-2,2.12Zm13.85-10H79.87a.37.37,0,0,0-.38.35h0V9.12A2.9,2.9,0,0,0,77,8a4.23,4.23,0,0,0-4.17,4.44,4.26,4.26,0,0,0,4.2,4.46,2.91,2.91,0,0,0,2.44-1.14v.53a.37.37,0,0,0,.36.37h1.74a.37.37,0,0,0,.37-.35V4.94a.35.35,0,0,0-.35-.36Zm-4,10a2,2,0,0,1-2-2.14,2,2,0,1,1,4,0A2,2,0,0,1,77.56,14.61Zm13.85-10H89.66a.37.37,0,0,0-.38.35h0V9.12A2.89,2.89,0,0,0,86.83,8a4.22,4.22,0,0,0-4.16,4.44,4.26,4.26,0,0,0,4.2,4.46,2.89,2.89,0,0,0,2.43-1.14v.53a.37.37,0,0,0,.36.37h1.75a.36.36,0,0,0,.36-.35V4.94a.35.35,0,0,0-.34-.36Zm-4.06,10a2,2,0,0,1-2-2.14,2,2,0,1,1,4,0,2,2,0,0,1-2,2.14ZM101,8.84l-2.6,8.93A3.33,3.33,0,0,1,95,20.31a4.21,4.21,0,0,1-2.08-.48c-.33-.19-.61-.36-.61-.64a.71.71,0,0,1,.16-.42L93,18c.15-.22.26-.29.41-.29a.55.55,0,0,1,.35.12,1.86,1.86,0,0,0,1.08.39A1.08,1.08,0,0,0,96,17.41l.23-.74H95.15a.46.46,0,0,1-.46-.36L92.36,8.84c-.09-.29,0-.58.42-.58h1.84c.22,0,.38.07.48.39l1.72,6,1.61-6a.47.47,0,0,1,.46-.39h1.74C101,8.25,101.08,8.5,101,8.84ZM40.8,11.5v4.77a.38.38,0,0,1-.38.39H39a.38.38,0,0,1-.39-.38h0V15a5.49,5.49,0,0,1-4.31,1.88,5.92,5.92,0,0,1-6-6.16,6.22,6.22,0,0,1,6.45-6.35c2.65,0,4.78,1.15,5.72,3.44a.45.45,0,0,1,0,.2c0,.1-.07.19-.3.27l-1.68.64a.4.4,0,0,1-.32,0,.5.5,0,0,1-.2-.23,3.52,3.52,0,0,0-3.33-1.93A3.67,3.67,0,0,0,31,10.64a3.58,3.58,0,0,0,3.74,3.83,3.47,3.47,0,0,0,2.78-1.26H36a.39.39,0,0,1-.39-.39h0V11.52a.38.38,0,0,1,.38-.4h4.46a.37.37,0,0,1,.39.36Zm60.21-5V6.35h.78v.16h-.29v.72a0,0,0,0,1,0,0h-.16V6.52H101a0,0,0,0,1,0,0Zm1.28-.12.2.46.2-.46,0,0H103a0,0,0,0,1,0,0h0v.87a0,0,0,0,1,0,0h-.15a0,0,0,0,1,0,0V6.59l-.21.47,0,0h-.16l0,0-.21-.47v.65a0,0,0,0,1,0,0H102a0,0,0,0,1,0,0V6.37a0,0,0,0,1,0,0h.25S102.29,6.36,102.29,6.37Z"/></svg>
```

**Pull request merged**